mdf_matio/adapters/basic_adapters.py

```diff
@@ -0,0 +1,60 @@
+from mdf_matio.adapters.generic import GenericMDFAdapter
+
+
+# Adapters that require no extra processing and can just use the GenericMDFAdapter
+
+class CrystalStructureAdapter(GenericMDFAdapter):
+    """Adapt the CrystalStructureExtractor"""
+    # No extra processing required
+    pass
+
+
+class ElectronMicroscopyAdapter(GenericMDFAdapter):
+    """Adapt the ElectronMicroscopyExtractor"""
+    # No extra processing required
+    pass
+
+
+class GenericFileAdapter(GenericMDFAdapter):
+    """Adapt the GenericFileExtractor"""
+    # No extra processing required
+    pass
+
+
+class FilenameAdapter(GenericMDFAdapter):
+    """Adapt the FilenameExtractor"""
+    # No extra processing required
+    pass
+
+
+class ImageAdapter(GenericMDFAdapter):
```

Comment:
We already have a image adapters listed as a "no op" parser. Do we need this?

Reply:
Same for generic

Reply:
Yes, this is better than a "no-op" adapter. The `GenericMDFAdapter` provides forward-compatibility, because MatIO extractors' output can only change by adding additional fields. Therefore, if all non-MDF-format fields are removed (by the `GenericMDFAdapter`), the adapters don't need to be changed if the extractor adds additional fields.

Reply:
Great, then could you remove the uses of `noop` parser? It seems like there isn't a reason we should use that one in place of this GenericMDFAdapter